app/javascript/admin/controllers/creator_settings_controller.js

```diff
@@ -45,23 +45,28 @@ export class CreatorSettingsController extends Controller {
             target: { width, height },
           } = event;
 
+          this.previewLogoTarget.replaceChild(
+            image,
+            this.previewLogoTarget.firstChild,
+          );
+
+          const maxLogoPreviewWidth = parseInt(
```

Comment:
Now the CSS var is read instead of a hard-coded width constant. This is done after the image is inserted into the DOM so that the CSS is applied and the CSS var can be read.